e2e/keywords/common.resource

```diff
@@ -70,32 +70,37 @@ Cleanup test resources
         Run keyword And Ignore Error    power_on_node_by_name    ${powered_off_node}
         Remove Values From List    ${powered_off_nodes}    ${powered_off_node}
     END
-    uncordon_all_nodes
-    cleanup_control_plane_network_latency
-    reset_node_schedule
-    cleanup_node_exec
-    cleanup_stress_helper
-    cleanup_recurringjobs
-    cleanup_deployments
-    cleanup_statefulsets
-    cleanup_pods
-    cleanup_persistentvolumeclaims
-    cleanup_volumes
-    cleanup_storageclasses
-    set_storageclass_default_state    longhorn    ${True}
-    cleanup_secrets
-    cleanup_backups
-    cleanup_system_backups
-    cleanup_system_restores
-    delete_orphans
-    cleanup_backing_images
-    cleanup_node_labels
-    cleanup_disks
-    cleanup_engine_images
-    reset_backupstore
-    reset_settings
-    reset_node_tags
-    reset_node_disks_tags
-    cleanup_k8s_node_zone
-    cleanup_csi_volume_snapshots
-    cleanup_csi_volume_snapshot_classes
+    TRY
```

Comment:
I think failures in resource cleanup should fail the test case, otherwise the leftover resources will silently pollute the environment and unexpectedly affect the subsequent test cases. If the subsequent test cases fail as a result, we will need more efforts to identify the issue.

Reply:
OK make sense. Just re-considered this part, returning an error in the teardown stage is better than silently polluting the next test.